bots/pr/src/main/java/org/openjdk/skara/bots/pr/SummaryCommand.java

```diff
@@ -50,17 +52,23 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
             var summary = command.args().lines()
                                  .map(String::strip)
                                  .collect(Collectors.joining("\n"));
-            var action = currentSummary.isPresent() ? "Updating existing" : "Setting";
-            if (summary.contains("\n")) {
-                reply.println(action + " summary to:\n" +
-                                      "\n" +
-                                      "```\n" +
-                                      summary +
-                                      "\n```");
+            if (!checkSummary(summary)) {
+                reply.println("\"" + summary + "\" is invalid, summary should not contain \"<BugId>:\", \"Co-authored-by:\", \"Reviewed-by:\" and \"Backport-of:\".\n" +
+                        "This Summary will be removed.");
```

Comment:
When quoting the given invalid summary please quote it in the same way as the accepting message below (multiline if needed, use back ticks). Prefix it with "Invalid summary: ". Then on the next line add the information about what cannot be included in a summary. I don't think we need the "This Summary will be removed." message on top of that.

Example:

Invalid summary:
```
foobar
12345: foo
```
A summary line cannot start with any of the following: `<issue-id>:`, `Co-authored-by:`, `Reviewed-by:`, `Backport-of:`. See [JEP 357](https://openjdk.org/jeps/357) for details.

Reply:
> When quoting the given invalid summary please quote it in the same way as the accepting message below (multiline if needed, use back ticks). Prefix it with "Invalid summary: ". Then on the next line add the information about what cannot be included in a summary. I don't think we need the "This Summary will be removed." message on top of that.
> 
> Example:
> 
> Invalid summary:
> 
> ```
> foobar
> 12345: foo
> ```
> 
> A summary line cannot start with any of the following: `<issue-id>:`, `Co-authored-by:`, `Reviewed-by:`, `Backport-of:`. See [JEP 357](https://openjdk.org/jeps/357) for details.

Thanks for the suggestion! I will change it.